Ductus.FluentDocker.Tests/AmbientContext/AmbientContext.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System;
+using System.Threading;
+using Ductus.FluentDocker.AmbientContex;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Ductus.FluentDocker.Tests.AmbientContext
+{
+  [TestClass]
+  public class AmbientContext
+  {
+    [TestMethod]
+    public void Test1000000Nested()
```

Comment:
Sorry, but what is this test, testing? Shouldn't it be any asserts in order to ensure that it contains the correct content?

Reply:
This was copied from the tests where I pulled the AmbientContext from. I can remove if you think that is required.

Reply:
please do